cla_backend/apps/cla_provider/tests/api/test_csvupload_api.py

```diff
@@ -605,6 +604,52 @@ def test_validate_in_iterable(self):
         with self.assertRaisesRegexp(serializers.ValidationError, r".*must be one of"):
             test_in("q")
 
+    @override_settings(CONTRACT_2018_ENABLED=True)
+    def test_validator_fafa_determination_code_is_valid(self):
```

Comment:
@Katy600 and @said-moj And the test from [Said's earlier FAFA branch](https://github.com/ministryofjustice/cla_backend/pull/492/files#diff-3ec79047d8d98a23737dee56339b5d0aR454).